src/app/portlets/dot-starter/dot-starter.component.scss

```diff
@@ -0,0 +1,191 @@
+@use "variables" as *;
+
+:host {
+    display: block;
+    height: 100%;
+    overflow: auto;
+
+    hr {
+        border-top: 1px solid $gray-lighter;
+    }
+
+    .p-col,
+    .p-col-fixed,
+    .p-grid {
+        margin: 0;
+        padding: 0;
+    }
+
+    a {
+        text-decoration: none;
+    }
+
+    .dot-starter-title {
+        color: $black;
+        font-size: $font-size-x-large;
+    }
+
+    .dot-starter-description {
+        color: $gray;
+        font-size: $font-size-large;
+    }
+
+    .dot-starter-top-content,
+    .dot-starter-bottom-content {
+        display: flex;
+        margin: 0 auto;
+        max-width: 1084px;
```

Comment:
I have to see it in the context of this component but why setting an absolute width here? What would happen in devices that its viewport is less than 1084px?

Reply:
this is to let the content expand up to a max width
![image](https://user-images.githubusercontent.com/37185433/101042242-95118880-3542-11eb-9208-1589b177301a.png)

when viewport is less than that it will be responsive

Reply:
rem